app/_locales/en/messages.json

```diff
@@ -32,20 +14,35 @@
   "confirmClear": {
     "message": "Are you sure you want to clear approved websites?"
   },
+  "connections": {
+    "message": "Connections"
+  },
+  "connectionsSettingsDescription": {
+    "message": "Sites allowed to read your accounts"
+  },
+  "addSite": {
+    "message": "Add Site"
+  },
+  "addSiteDescription": {
+    "message": "Manually add a site to allow it access to your accounts, useful for older dapps"
+  },
+  "connected": {
+    "message": "Connected"
+  },
+  "connectedDescription": {
+    "message": "The list of sites allowed access to your addresses"
+  },
+  "privacyModeDefault": {
+    "message": "Privacy Mode is now enabled by default"
+  },
   "contractInteraction": {
     "message": "Contract Interaction"
   },
   "clearApprovalDataSuccess": {
     "message": "Approved website data cleared successfully."
   },
-  "approvalData": {
```

Comment:
These two messages seem to remain present in most locales as well

Reply:
Good catch, removed